toc.json

```diff
@@ -47,7 +47,8 @@
       "cdns": "Packages from CDNs",
       "import_maps": "Using import maps",
       "faqs": "Frequently asked questions",
-      "cheatsheet": "Node->Deno cheatsheet"
+      "cheatsheet": "Node->Deno cheatsheet",
+      "dnt": "dnt"
```

Comment:
Better title? I couldn't think of anything short.

Reply:
`dnt - Deno to Node Transform`?  I think it should be something more than just "dnt" because almost everyone would be huh?